src/lib/swapper/swappers/ThorchainSwapper/buildThorTrade/buildThorTrade.ts

```diff
@@ -113,13 +113,17 @@ export const buildTrade = async (
     if (maybeThorTxInfo.isErr()) return Err(maybeThorTxInfo.unwrapErr())
     const { vault, opReturnData } = maybeThorTxInfo.unwrap()
 
-    // TODO(gomes): it is not - plumb from through
```

Comment:
Actually is to get from, but doesn't need to be passed down - that's why we have `buildSignTx`